django-verdant/rca_ee/models.py

```diff
@@ -107,10 +107,16 @@ def serve(self, request):
         context['form'] = form
         return render(
             request,
-            self.template,
+            self.get_template(request),
             context
         )
 
+    def get_template(self, request, *args, **kwargs):
+        if request.is_ajax() and request.GET.get('format') == 'enquiry_form':
```

Comment:
Curiously, the form page already has a `Vary:X-Requested-With` header but I don't see where that's being added. Nonetheless, it would be good to add `@vary_on_headers('X-Requested-With')` to the serve method just to make sure this will always be the case.